lib/ansible/modules/net_tools/netplan.py

```diff
@@ -0,0 +1,903 @@
+        specific/desirable/nicer name than the default from udev's ifnames.
+    required: false
+
+  wakeonlan:
+    description:
+      - Enable wake on LAN. Off by default.
+    required: false
+    type: bool
+
+  dhcp4-overrides-use-dns:
+    description:
+      - The DNS servers received from the DHCP server will be used and take
+        precedence over any statically. Only C(networkd) backend and C(dhcp4)
+        must be true.
+    required: false
+    type: bool
+
+  dhcp4-overrides-use-ntp:
+    description:
+      - The NTP servers received from the DHCP server will be used by
+        systemd-timesyncd and take precedence over any statically configured
+        ones. Only C(networkd) backend and C(dhcp4) must be true.
+    required: false
+    type: bool
+
+  dhcp4-overrides-use-hostname:
+    description:
+      - The hostname received from the DHCP server will be set as the transient
+        hostname of the system. Only C(networkd) backend and C(dhcp4)
+        must be true.
+    required: false
+    type: bool
+
+  dhcp4-overrides-send-hostname:
+    description:
+      - The machine's hostname will be sent to the DHCP serverself.
+        Only C(networkd) backend and C(dhcp4) must be true.
+    required: false
+    type: bool
+
+  dhcp4-overrides-hostname:
+    description:
+      - Use this value for the hostname which is sent to the DHCP server,
+        instead of machine's hostname. Only C(networkd) backend and C(dhcp4)
+        must be true.
+    required: false
+
+  dhcp6-overrides-use-dns:
+    description:
+      - The DNS servers received from the DHCP server will be used and take
+        precedence over any statically. Only C(networkd) backend and C(dhcp6)
+        must be true.
+    required: false
+    type: bool
+
+  dhcp6-overrides-use-ntp:
+    description:
+      - The NTP servers received from the DHCP server will be used by
+        systemd-timesyncd and take precedence over any statically configured
+        ones. Only C(networkd) backend and C(dhcp6) must be true.
+    required: false
+    type: bool
+
+  dhcp6-overrides-use-hostname:
+    description:
+      - The hostname received from the DHCP server will be set as the transient
+        hostname of the system. Only C(networkd) backend and C(dhcp6)
+        must be true.
+    required: false
+    type: bool
+
+  dhcp6-overrides-send-hostname:
+    description:
+      - The machine's hostname will be sent to the DHCP serverself.
+        Only C(networkd) backend and C(dhcp6) must be true.
+    required: false
+    type: bool
+
+  dhcp6-overrides-hostname:
+    description:
+      - Use this value for the hostname which is sent to the DHCP server,
+        instead of machine's hostname. Only C(networkd) backend and C(dhcp6)
+        must be true.
+    required: false
+
+  interfaces:
+    description:
+      - All devices matching this ID list will be added or associated to
+        bridges or bonds.
+    required: false
+    type: list
+
+  mode:
```

Comment:
Rename to something more specific.  Perhaps `bonding-mode`, or `aggregation-mode`.